packages/itwin/tree-widget/src/components/trees/stateless/categories-tree/CategoriesTreeComponent.tsx

```diff
@@ -79,19 +80,21 @@ function CategoriesTreeComponentImpl({
                 />,
               ]}
         </TreeHeader>
-        <AutoSizer>
-          {({ width, height }) => (
-            <StatelessCategoriesTree
-              {...treeProps}
-              imodel={iModel}
-              categories={categories}
-              activeView={viewport}
-              width={width}
-              height={height}
-              filter={filter}
-            />
-          )}
-        </AutoSizer>
+        <div className={classNames("tree-widget-tree-content", density === "enlarged" && "enlarge")}>
```

Comment:
I see this uses css classes from https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/tree-widget/src/components/trees/common/TreeRenderer.scss which includes quite a lot of css used for old tree components. Can you add a new cleaned up scss file under `stateless` folder that could be shared between stateless trees with header? This should make everything simpler when we will be removing old tree implementations